src/bridge/memory_util/openssl_bio_test.cc

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+#include <openssl/bio.h>
+#include <openssl/pem.h>
+
+#include "src/bridge/memory_util/openssl_bio.h"
+#include "src/testing_util/test_matchers.h"
+#include "src/testing_util/openssl_assertions.h"
+
+namespace kmsengine {
+namespace bridge {
+namespace {
+
+using ::testing::StrEq;
+using ::kmsengine::testing_util::IsOk;
+
+constexpr char kRsaPublicKey[] = "-----BEGIN PUBLIC KEY-----\n"
```

Comment:
There isn't anything PEM-specific about BIOs, right? Maybe make the buffer contents generic?